src/DacpacTool/ScriptParser.cs

```diff
@@ -24,8 +25,20 @@ private void Parse()
         {
             if (! _parsed)
             {
-                _parser.Parse();
-                _parsed = true;
+                try
+                {
+                    _parser.Parse();
+                    _parsed = true;
+                }
+                catch (Microsoft.SqlTools.ServiceLayer.BatchParser.BatchParserException ex)
```

Comment:
Do we need to fully qualify the exception type? There's already a using for this namespace in this file?